manuscript/01.introduction.md

```diff
@@ -11,8 +11,8 @@ Large-scale profiling studies can find molecular features that associate with dr
 Measurements accompanied by lineage relationships are uniquely valuable for studying inherited phenotypes within families of people or populations of cells. This value is evident in linkage studies wherein relatives are used to identify or refine the genetic determinants of disease [@DOI:10.1126/science.aax3710; @DOI:10.1038/s41467-020-17117-4; @pmid:19136655]. Notably, linkage studies can identify genetic determinants with greater power than even large association studies because relatives essentially serve as internal controls [@PMID:19369670]. Linkage studies also start with the phenotype of individuals, rather than grouping based on molecular differences. Defining heterogeneity phenotypically has previously helped to identify subpopulations that would be hidden if using molecularly-defined clusters [@DOI:10.1016/j.isci.2020.101991]. Lineage-resolved data has also demonstrated particular value in uncovering heterogeneity due to transient differences [@doi:10.1101/373258; @doi:10.1073/pnas.1715639115].
 
 <!-- tHMMs are a solution to modeling lineage data -->
-Hidden Markov models (HMMs) provide an efficient strategy to infer discrete states from measurements when a series of co-dependent observations are made. An example of this is their wide-spread use in time series analysis, where each measurement is dependent on those that came before (CITE). Recognizing this co-dependence allows HMMs to make accurate inferences even in the presence of extremely noisy measurements since each neighboring measurement provides accumulating evidence for a prediction [@DOI:10.1126/scisignal.aao1917]. HMMs have been adapted to lineage trees (tHMMs) so that each measurement across the tree can similarly provide accumulating evidence for a prediction. Just like with time-series data, these models can provide very accurate predictions despite noisy measurements and limited information by recognizing the co-dependence between related measurements [@doi:10.1109/78.668544; @doi:10.1109/TSP.2004.832006]. tHMMs have been used in a multitude of applications, from image classification to comparative genomics [@pmid:18255546; @pmid:23762278]. In cells, these models have been fit to lineages collected from stem cells and bacteria colonies but have always required tailor-made implementations [@PMID:19628503; @doi:10.1101/488981]. Improvements in cell tracking and high-throughput imaging promise to make these models valuable techniques for studying the plasticity of heterogeneous cell populations. However, widespread use will require usable implementations that can be readily adapted to different experimental measurements, examples of unique insights revealed, and standards for experimental validation.
```

Comment:
Why this paper?

Reply:
It talks about the application of HMM for time series data... Isn't it the point?

Reply:
Should I replace this?

Reply:
This is a poor example. Surely there is a more expansive paper or review that covers this topic in general, like this sentence is meant to address.

Reply:
I have a couple of papers open in my browser for 3 days and I have not been able to finish reading them. I returned this to `(CITE)` so I can later choose the best match literature to it.